python/ray/autoscaler/command_runner.py

```diff
@@ -484,13 +485,15 @@ def run(
 
     def run_rsync_up(self, source, target):
         self._set_ssh_ip_if_required()
```

Comment:
Could you possibly include this as an argument to `run_rsync_up` and `run_rsync_down` instead of the constructors?